src/librc/librc-depend.c

```diff
@@ -682,14 +692,17 @@ rc_deptree_update_needed(time_t *newest, char *file)
 	time_t mtime;
 
 	/* Create base directories if needed */
-	for (i = 0; depdirs[i]; i++)
-		if (mkdir(depdirs[i], 0755) != 0 && errno != EEXIST)
-			fprintf(stderr, "mkdir `%s': %s\n", depdirs[i], strerror(errno));
+	char depdir[sizeof(char) * PATH_MAX];
```

Comment:
Aside from `sizeof(char)` being 1, this is still pretty nonsensical. Arrays are declared in amount of elements, not bytes. So `int v[5]` will allocate enough space for 5 ints.

Reply:
Fixed.